src/wp-admin/options-general.php

```diff
@@ -120,8 +120,11 @@
 	}
 
 	// Handle alt text for site icon on page load.
-	$img_alt_value = '';
-	$site_icon_id  = (int) get_option( 'site_icon' );
+	$img_alt_value           = '';
+	$site_icon_id            = (int) get_option( 'site_icon' );
+	$app_icon_alt_prefix     = _x( 'App icon preview', 'Prefix for alternative text of the site icon preview' );
+	$browser_icon_alt_prefix = _x( 'Browser icon preview', 'Prefix for alternative text of the site icon preview' );
```

Comment:
Ideally, the context parameter of `_x()` should only contain very short text to clarify the context e.g. verb, name, or the object the string refers to e.g. ctegories, tags, etc. See usages in core. The actual usage of this parameter in Gutenberg is sometimes not ideal, please refer to core instead.